src/validator/LookupValidator.cpp

```diff
@@ -388,6 +388,12 @@ Status LookupValidator::rewriteRelExpr(RelationalExpression* expr) {
 }
 
 StatusOr<Value> LookupValidator::checkConstExpr(Expression* expr, const std::string& prop) {
+    // TODO (sky) : just check the const expression now.
+    // other expressions will be supported in the future. such as :
+    // col1 > 1 + 2 , col1 > abs(const) , etc.
+    if (expr->kind() != Expression::Kind::kConstant) {
+        return Status::SemanticError("expression error : %s", expr->toString().c_str());
+    }
```

Comment:
we could use `evaluableExpr` here.

Reply:
> we could use `evaluableExpr` here.

Thanks, this is what I really want. fixed.